### PR TITLE
Twig 3

### PR DESCRIPTION
**Pull request by andrew-nuwber**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | yes
| Tests pass?   | yes
| Fixed issues  |

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.2.5",
         "yiisoft/yii2": "~2.0.4",
-        "twig/twig": "~2.7"
+        "twig/twig": "~3.0"
```
Comment (schmunk42) [inline composer.json]: We should release at least `2.4.x` or even `3.x` after merge.

Comment (samdark) [reply composer.json]: 2.4.0, I think.

**Comment by masterjus** [comment]
We found that now pages generating a bit faster. May anybody to force this PR approval?

**Comment by samdark** [comment]
Yes, it's on my list...

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Comment by andrew-nuwber** [comment]
Could you also update the package in composer? 
https://github.com/yiisoft/yii2-twig/pull/123

**Comment by samdark** [comment]
@andrew-nuwber what do you mean? It's updated already https://github.com/yiisoft/yii2-twig/blob/master/composer.json#L23

**Comment by andrew-nuwber** [comment]
@samdark there is no version 2.4.0 in [packagist](https://packagist.org/packages/yiisoft/yii2-twig)

**Comment by samdark** [comment]
Oh, weird. I need to check hooks. Updated manually for now.

**Pull request merged**